target/bin/fail2ban

```diff
@@ -3,7 +3,11 @@
 # shellcheck source=../scripts/helpers/index.sh
 source /usr/local/bin/helpers/index.sh
 
-function __usage { echo "Usage: ${0} [<unban> <ip-address>]" ; }
+function __usage {
+  echo "Usage: ${0}" ;
+  echo "       ${0} ban <IP>" ;
+  echo "       ${0} unban <IP>" ;
```

Comment:
Why the `;` at the end? Seems unnecessary to me.

Reply:
full ack. copy/paste error.